Library/Homebrew/extend/os/ENV/super.rb

```diff
@@ -0,0 +1,5 @@
+require "ENV/super"
```

Comment:
Yeh, `extend/os/extend` felt weird but happy to use that if it makes more sense to you.